scripts/check_format.sh

```diff
@@ -41,23 +41,23 @@ function check_shogun_style {
             fi
         done
 
-        RESULT_OUTPUT="$(git clang-format-3.8 --commit $BASE_COMMIT --diff --binary `which clang-format-3.8` $LIST)"
+        RESULT_OUTPUT="$(git clang-format-3.9 --commit $BASE_COMMIT --diff --binary `which clang-format-3.9` $LIST)"
 
         if [ "$RESULT_OUTPUT" == "no modified files to format" ] \
-            || [ "$RESULT_OUTPUT" == "clang-format-3.8 did not modify any files" ] \
+            || [ "$RESULT_OUTPUT" == "clang-format-3.9 did not modify any files" ] \
             || [ "$RESULT_OUTPUT" == "clang-format did not modify any files" ]; then
-              echo "clang-format-3.8 passed. \o/"
+              echo "clang-format-3.9 passed. \o/"
               echo "-----"
               exit 0
         else
             echo "-----"
             echo "clang-format failed."
             echo "To reproduce it locally please run: "
             echo -e "\t1) git checkout ${1:-}"
-            echo -e "\t2) git clang-format-3.8 --commit $BASE_COMMIT --diff --binary $(which clang-format-3.8)"
+            echo -e "\t2) git clang-format-3.9 --commit $BASE_COMMIT --diff --binary $(which clang-format-3.9)"
             echo "To fix the errors automatically please run: "
             echo -e "\t1) git checkout ${1:-}"
-            echo -e "\t2) git clang-format-3.8 --commit $BASE_COMMIT --binary $(which clang-format-3.8)"
+            echo -e "\t2) git clang-format-3.9 --commit $BASE_COMMIT --binary $(which clang-format-3.9)"
```

Comment:
Why did you change the clang-format version?

Reply:
I tried to install 3.8 version on ubuntu 18.04 but couldn't find any way to do it. I did a while back and so I am not really able to recall the details now. So, I finally decided to change it to 3.9 version which is available in 18.04

Reply:
oh right, I think the 3.8 version is the one used in the Dockerfile, so I don't think you should change it here. You can use any other version for your own setup and then run it on a specific file using the .clang-format file in shogun :)

Reply:
Okay. I will undo the change. Thanks :+1: